### PR TITLE
Four Linux Question added

### PR DESCRIPTION
**Pull request by GulabSinghSikarwar**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -759,6 +759,15 @@ const linuxQuiz = [
...
     Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/#the-linux-kill-command",
   },
+  {
+    Question: "OSS stands for _____",
+    Answer: "Open Source Software",
+    Distractor1: "Original Source Software",
+    Distractor2: "Open System Software",
+    Distractor3: "Original System Server",
+    Explanation: "OSS stands for Open Source Software.",
+    Link: "https://www.freecodecamp.org/news/what-is-open-source-software-explained-in-plain-english/",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:769]: This isn't really a Linux specific question, nor does it really fit in any of the existing categories.
Let's just remove it

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -769,6 +778,37 @@ const linuxQuiz = [
...
+    Answer: "superuser do",
+    Distractor1: "su",
+    Distractor2: "superuser does",
+    Distractor3: "super do",
+    Explanation: "The sudo command stands for superuser do.",
+    Link: "https://www.freecodecamp.org/news/the-ultimate-guide-to-linux-creating-a-sudo-user/",
+  },
+  {
+    Question:
+      "Which is the correct command syntax to rename a file abc.txt to pqr.txt",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:792]:
```suggestion
      "In Linux, which one is the correct syntax for renaming a file called abc.txt to pqr.txt?",
```

File: src/data/linux-quiz.ts [hunk 3]
```diff
@@ -769,6 +778,37 @@ const linuxQuiz = [
...
+    Answer: "mv abc.txt pqr.txt",
+    Distractor1: "mv pqr.txt abc.txt",
+    Distractor2: "mv -a pqr.txt abc.txt",
+    Distractor3: "mv -all pqr.txt abc.txt",
+    Explanation:
+      "The correct command syntax to rename a file abc.txt to pqr.txt is: mv abc.txt pqr.txt",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:798]:
```suggestion
      "The correct syntax for renaming a file called abc.txt to pqr.txt is mv abc.txt pqr.txt",
```

File: src/data/linux-quiz.ts [hunk 4]
```diff
@@ -769,6 +778,37 @@ const linuxQuiz = [
...
+    Link: "https://www.freecodecamp.org/news/rename-file-linux-bash-command/",
+  },
+  {
+    Question:
+      "Which Linux command is used to go back from a folder to the folder before that?",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:803]:
```suggestion
      "Which Linux command is used to go up one level to a parent folder?",
```

File: src/data/linux-quiz.ts [hunk 5]
```diff
@@ -769,6 +778,37 @@ const linuxQuiz = [
...
+    Answer: "cd..",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:804]:
```suggestion
    Answer: "cd ..",
```

File: src/data/linux-quiz.ts [hunk 6]
```diff
@@ -769,6 +778,37 @@ const linuxQuiz = [
...
+    Distractor1: "cd",
+    Distractor2: "cd/",
+    Distractor3: "chdir",
+    Explanation:
+      ".. is a special path used to indicate the parent folder and using cd .. command you can go back from a folder to the folder before that ",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:809]:
```suggestion
      ".. is a special path used to indicate the parent folder. cd .. is used to go up one level to a parent folder.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by GulabSinghSikarwar** [comment]
Thanks, @jdwilkin4  for reviewing, I will indeed work on the changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**